index.html

```diff
@@ -0,0 +1,2059 @@
+                  class="card-img-top"
+                  alt="walmart-store-front"
+                />
+              </div>
+              <div class="card-body">
+                <h5 class="card-title font-weight-bold">
+                  <a class="h5-a" href="#"
+                    >Newsweek AMPLIFY Launches Strategic Partnership Solutions
+                    for Brands</a
+                  >
+                </h5>
+                <p class="card-text">
+                  Newsweek AMPLIFY, the company's performance marketing business
+                  unit, has launched a new program to empower digital
+                </p>
+              </div>
+            </article>
+          </div>
+          <!--End of survey & rankings section-->
+          <!--More Stories section-->
+          <section class="d-md-none gap">
+            <div class="more-stories mt-5">
+              <div class="h2-div font-weight-bold">MORE STORIES</div>
+              <hr />
+              <div class="media-1">
+                <img
+                  class="float-left mr-3"
+                  alt="64x64"
+                  src="https://d.newsweek.com/en/full/1626297/joe-biden-jill-dnc-2020.jpg?w=95&h=95&f=44299f0d2be506ecefd4079ee9206bdf"
+                  style="width: 95px; height: 95px;"
+                />
+                <div class="media-body">
+                  <h5 class="mt-0 font-weight-bold">
+                    DNC Night 4 Speeches and Highlights: Biden, Atlanta Mayor
+                    and 'Veep' Star
+                  </h5>
+                  <p>
+                    The Democratic presidential nominee ended the mostly-virtual
+                    DNC celebrations with a speech calling for national unity.
+                  </p>
+                </div>
+              </div>
+              <div class="media-1">
+                <img
+                  class="float-left mr-3"
+                  alt="64x64"
+                  src="https://d.newsweek.com/en/full/1626322/goodyear.jpg?w=95&h=95&f=46d89aea47cb989d16a3f626b6a66d4c"
+                  style="width: 95px; height: 95px;"
+                />
+                <div class="media-body">
+                  <h5 class="mt-0 font-weight-bold">
+                    Goodyear Says 'Blue Lives Matter' Ban a Mistake After Trump
+                    Boycott Call
+                  </h5>
+                  <p>
+                    Company said it is still asking emplyees to "refrain" from
+                    political endoresements but will allow support for law
+                    enforcement.
+                  </p>
+                </div>
+              </div>
+              <div class="media-1">
+                <img
+                  class="float-left mr-3"
+                  alt="64x64"
+                  src="https://d.newsweek.com/en/full/1626230/bidens-granddaughters.jpg?w=95&h=95&f=dcaa9568f1a7cecc93620cb3c6fd0d16"
+                  style="width: 95px; height: 95px;"
```

Comment:
- [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)